Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -43,21 +50,23 @@ export default function useAndroidRippleForView(
     if (
       Platform.OS === 'android' &&
       Platform.Version >= 21 &&
-      rippleColor != null
+      rippleConfig != null
     ) {
-      const processedColor = processColor(rippleColor);
+      const {color, borderless, radius} = rippleConfig;
+      const processedColor = processColor(color);
       invariant(
         processedColor == null || typeof processedColor === 'number',
         'Unexpected color given for Ripple color',
       );
 
       return {
         viewProps: {
-          // Consider supporting `nativeForegroundAndroid` and `borderless`.
+          // Consider supporting `nativeForegroundAndroid`
           nativeBackgroundAndroid: {
             type: 'RippleAndroid',
             color: processedColor,
-            borderless: false,
+            borderless: !!borderless,
```

Comment:
I think we prefer being explicit:

borderless: borderless === true

Reply:
did that, marking as resolved